fix: gradle plugin extra config (#2533)
getsentry/sentry-docs: src/platforms/android/migration.mdx

```java
Sentry.setExtra("extra", "thing");
```

### Sentry Gradle Plugin

> Disable the `autoProguardConfig` flag from the Sentry Gradle Plugin, `sentry-android` >= v2.0 already takes care of it automatically.

Author: ```suggestion
Disable the `autoProguardConfig` flag from the Sentry Gradle Plugin, `sentry-android` >= v2.0 already does it automatically.
```
